tests/test_form_data.py

```diff
@@ -0,0 +1,17 @@
+from app import app
+
+
```

Comment:
nit: unnecessary blank lines

Reply:
Unfortunately It's pythons weird style guide!